openai-functions-codeinterpreter/requirements.txt

```diff
@@ -0,0 +1,87 @@
+aiofiles==23.1.0
```

Comment:
you can replace a lot of these dependencies with just a `chainlit` dependency. Most of them are dependencies of Chainlit